Src/Newtonsoft.Json-for-Unity.Tests/Assets/SystemWebPolyfill/HttpUtility.cs

```diff
@@ -0,0 +1,670 @@
+        }
+
+        public static string UrlEncodeUnicode (string str)
+        {
+            if (str == null)
+                return null;
+
+            return Encoding.ASCII.GetString (UrlEncodeUnicodeToBytes (str));
+        }
+
+        public static byte [] UrlEncodeUnicodeToBytes (string str)
+        {
+            if (str == null)
+                return null;
+
+            if (str.Length == 0)
+                return new byte [0];
+
+            MemoryStream result = new MemoryStream (str.Length);
+            foreach (char c in str){
+                HttpEncoder.UrlEncodeChar (c, result, true);
+            }
+            return result.ToArray ();
+        }
+
+        /// <summary>
+        /// Decodes an HTML-encoded string and returns the decoded string.
+        /// </summary>
+        /// <param name="s">The HTML string to decode. </param>
+        /// <returns>The decoded text.</returns>
+        public static string HtmlDecode (string s) 
+        {
+            if (s == null)
+                return null;
+            
+            using (var sw = new StringWriter ()) {
+                HttpEncoder.Current.HtmlDecode (s, sw);
+                return sw.ToString ();
+            }
+        }
+    
+        /// <summary>
+        /// Decodes an HTML-encoded string and sends the resulting output to a TextWriter output stream.
+        /// </summary>
+        /// <param name="s">The HTML string to decode</param>
+        /// <param name="output">The TextWriter output stream containing the decoded string. </param>
+        public static void HtmlDecode(string s, TextWriter output) 
+        {
+            if (output == null) {
+                throw new ArgumentNullException ("output");
+            }
+                
+            if (!String.IsNullOrEmpty (s)) {
+                HttpEncoder.Current.HtmlDecode (s, output);
+            }
+        }
+
+        public static string HtmlEncode (string s)
+        {
+            if (s == null)
+                return null;
+            
+            using (var sw = new StringWriter ()) {
+                HttpEncoder.Current.HtmlEncode (s, sw);
+                return sw.ToString ();
+            }
+        }
+        
+        /// <summary>
+        /// HTML-encodes a string and sends the resulting output to a TextWriter output stream.
+        /// </summary>
+        /// <param name="s">The string to encode. </param>
+        /// <param name="output">The TextWriter output stream containing the encoded string. </param>
+        public static void HtmlEncode(string s, TextWriter output) 
+        {
+            if (output == null) {
+                throw new ArgumentNullException ("output");
+            }
+                
+            if (!String.IsNullOrEmpty (s)) {
+                HttpEncoder.Current.HtmlEncode (s, output);
+            }
+        }
+        public static string HtmlEncode (object value)
+        {
+            if (value == null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4755183)